NoPythonDemo/StyleKitSample/Style.swift

```diff
@@ -70,35 +70,43 @@ class Style: NSObject {
     
     var styleMap = [UIElement:StyleMap]()
     
+    enum StyleKitError: ErrorType {
```

Comment:
We already have 'ParseError: ErrorType' but I like your name better, so I think we should merge the enums found in ParseError into this?